modules/api.js

```diff
@@ -23,17 +23,15 @@ import { createPlatesFromBarcodes } from './lighthouse_service'
 const createSamples = async (boxBarcode) => {
   const platesForBoxBarcode = await getPlatesFromBoxBarcodes([boxBarcode])
 
-  if (platesForBoxBarcode.length === 0) {
+  if (!platesForBoxBarcode.success) {
     return [
       {
         errors: [`Failed to get plate barcodes for box barcode: ${boxBarcode}`]
       }
     ]
   }
 
-  const response = await createPlatesFromBarcodes({
-    plateBarcodes: platesForBoxBarcode
-  })
```

Comment:
Is this intentional, changing argument from object to single argument?
I can't see a corresponding change in the lighthouse service class?

Reply:
Is it still working ok?

Reply:
It's fine. plateBarcodes is still part of module options.

Reply:
ah, ok, because now we're adding the key in labwhere.js rather than here